### ISSUE TITLE
Principles of this repo

### ISSUE THREAD

**Issue by jodastephen** [JodaOrg/global-tz#1]
This is a discussion issue for how we can successfully fork [`eggert/tz`](https://github.com/eggert/tz).

Since everyone involved is likely to have limited time to maintain the project, I propose that it operates primarily as an automated job that copies and manipulates data from `eggert/tz`. The result would be committed to this repo in a subfolder.

The project would operate on the basis that it provides a timezone ID for each inhabited region of the Earth where clocks have differed since 1970, with additional IDs added to ensure that there is at least one ID for each inhabited ISO-3166-2 code. Each ID will contain the best available information on the timezone history of that location (pre and post 1970) . Additional aliases will be included as has been customary in `eggert/tz`.

In other words, this is not intended to be a fork that operates in an entirely different way. Instead it is intended to be a compatible restatement of the data in `eggert/tz` with the policy reverted to that in use prior to 2014.

Comments and thoughts?
